src/Redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState: {
+    items: [],
+  },
+  reducers: {
+    addBook: (state, action) => {
+      state.items.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.items = state.items.filter((book) => book.id !== action.payload);
+    },
+  },
+});
+
```

Comment:
- [ ] Please consider exporting the `booksSlice.actions` in your case are `addBook` and `removeBook` 👍🏼 so you can use them when an  event is triggered in your components in the next project 👏🏼 

Imagine a scenario where you have `addPotato` and `addTomato` action creators with a `vegetablesSlice` slice so exporting the action creators will be something like:
```javascript
const { addPotato, addTomato } = vegetablesSlice.actions;
```